src/__tests__/index.test.ts

```diff
@@ -276,6 +276,11 @@ const vectors: Array<TestVector> = [
     coinType: 239,
     passingVectors: [{ text: 'AXaXZjZGA3qhQRTCsyG5uFKr9HeShgVhTF', hex: '17ad5cac596a1ef6c18ac1746dfd304f93964354b5' }],
   },
+  {
+    name: 'GRIN',
+    coinType: 250,
```

Comment:
coinType 250 belongs to SOOM

https://github.com/satoshilabs/slips/blob/master/slip-0044.md